lib/screenplay/jobs/takeover_tool_testing_job.ex

```diff
@@ -0,0 +1,137 @@
+defmodule Screenplay.Jobs.TakeoverToolTestingJob do
+  @moduledoc """
+  Module that executes automated testing for the OFM Takeover Tool.
+  Job tests two things:
+  1. We have the ability to write to and delete from the test folder `ZZZ-MBTA-TEST`
+  2. We see all station folders that we expect to see depending on the screens located at the station.
+  """
+
+  alias Screenplay.Outfront.SFTP
+
+  require Logger
+
+  @test_sftp_directory_name "ZZZ-MBTA-TEST"
+
+  def run do
+    conn = start_connection()
+
+    try do
+      test_creating_and_removing_images(conn)
+      all_directories_exist?(conn)
+    rescue
+      e ->
+        message = "[takeover_tool_testing sftp_connection_error] #{inspect(e)}"
+        Logger.error(message)
+        Sentry.capture_message(message, level: "error")
+    after
+      sftp_client_module().disconnect(conn)
+    end
+  end
+
+  # sobelow_skip ["Traversal.FileModule"]
+  defp test_creating_and_removing_images(conn) do
+    local_image_path = Path.join(:code.priv_dir(:screenplay), "takeover_test.png")
+    local_image_data = File.read!(local_image_path)
+
+    Enum.each(["Portrait", "Landscape"], fn orientation ->
```

Comment:
We use these specific directory names both here and I think twice in `all_directories_exist` — could we extract them to constants? It's a small thing, but I'd want to make sure that if Outfront decides to rename these, we can change the name in one place and be confident we got everything.

```elixir
@landscape_dir "Landscape"
@portrait_dir "Portrait"
```

Following that train of thought... are these also hard-coded in the "real" tool?

Reply:
They are hard-coded, yes. OFM's SFTP directory structure is `./Landscape/...` and `./Portrait/...`.

Reply:
I think I misunderstood what you meant by "real" tool for some reason. They were indeed hardcoded over there are well. I made those module constants as well, but am not a fan of having two sets of constants that refer to the same thing. What would be the best way for these two modules to share these constants? App config maybe?

Reply:
Maybe that new config file you extracted, since we are already storing some other directory names in there? They could then still be module constants, just loaded with `compile_env!`.